src/index.ts

```diff
@@ -90,39 +178,13 @@ export default async function (manager: Manager, settings: ComponentSettings) {
     handler('pagevisit', event, settings)
   })
 
-  manager.addEventListener('addtocart', event => {
```

Comment:
Sorry, I'm not sure I understand, @ad-astra-via but why are we removing these events? Some MCs still use these I believe.. Will we have to migrate those to some other event? If yes then which event?

Reply:
Instead of removing these completely, we can refactor it (to save some lines of code) by iterating of an array of the event names instead.

Something like:
```typescript
  ['addtocart', 'purchase', 'somethingElse'].forEach(ev => {
    manager.addEventListener(ev, event => {
      handler(ev, event, settings)
    })  
  })
  ```